content/blog/20240808-stac-1.1.0-beta.1.md

```diff
@@ -0,0 +1,237 @@
+After some intense time of discussions and document editing, we are proud to announce the release of **[STAC 1.1.0-beta.1](https://github.com/radiantearth/stac-spec/releases/tag/v1.1.0-beta.1)**.
+
+The focus has been the addition of a [common band construct](#bands) to unify the fields `eo:bands` and `raster:bands`.
+Additionally, [Item Asset Definition](#item-asset-defintions) (field `item_assets`) - formerly a popular STAC extension - is now part of the core specification.
+Various [additional fields](#common-metadata) have been made available via the common metadata mechanism, e.g. `keywords`, `roles`, `data_type` and `unit`.
+We collaborated closely with the editors of OGC API - Records to align better with STAC, which resulted in a change to the [`license`](#license) field for example.
+The [link object](#links) was extended to support additional HTTP mechanisms such as HTTP methods other than `GET` and HTTP headers.
+The [best practices](#best-practices) have evolved and various minor changes and clarifications were integrated throughout the specification.
+
+This is a beta release with the option to make changes before a final 1.1.0 release if the feedback of the STAC community asks for it in the next weeks. Currently, no further changes are planned at the time of this release though. You can find the [open issues for 1.1.0](https://github.com/radiantearth/stac-spec/milestone/33) in the GitHub issue tracker if any get submitted in the next weeks.
+
+We'd appreciate it if the STAC community takes some time to implement the new version and give feedback via the [GitHub issue tracker](https://github.com/radiantearth/stac-spec/issues) so that we can ensure the new version is mature and well-received by the community. We hope that the STAC ecosystem catches up with the changes in the specification in the next weeks and months, any help that you can provide is highly appreciated. If you want to fund the STAC work we do, please get in touch, too.
+
+A shoutout to all the participants and sponsors of the last STAC sprint in Philadelphia, who laid a solid basis for this release.
+Emmanuel Mathot and I were then funded by the STAC PSC to finalize the work. Thank you to everyone who made this possible.
+
+Please read the [Changelog](https://github.com/radiantearth/stac-spec/blob/master/CHANGELOG.md#v110-beta1---2024-08-08) for all changes that have been made to the specification since v1.0.0. In the following sections, we'll highlight the most important changes in the specification with some JSON snippets.
+
+## Bands
+
+As of STAC 1.1, the `bands` array merges the similar but separate fields `eo:bands` and `raster:bands`, which was probably one of the most annoying things in STAC for [historical reasons](https://github.com/radiantearth/stac-spec/discussions/1213). The new `bands` field can be used in combination with property inheritance to provide users with more flexibility.
```

Comment:
> property inheritance

This is the first time inheritance is mentioned, we should define it or link to a definition.

Reply:
```suggestion
As of STAC 1.1, the `bands` array merges the similar but separate fields `eo:bands` and `raster:bands`, which was probably one of the most annoying things in STAC for [historical reasons](https://github.com/radiantearth/stac-spec/discussions/1213). The new `bands` field can be used in combination with [property inheritance](https://github.com/radiantearth/stac-spec/blob/master/commons/assets.md#additional-fields) to provide users with more flexibility.
```